inference-engine/tests/ngraph_functions/src/squeeze_unsqueeze.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+std::shared_ptr<ngraph::Node> makeSqueezeUnsqueeze(const ngraph::Output<Node> &in,
+                                                   const element::Type &type,
+                                                   const std::vector<int> &squeeze_indices,
+                                                   ngraph::helpers::SqueezeOpType opType,
+                                                   bool isScalarAxes) {
+    ngraph::Shape shape = isScalarAxes ? ngraph::Shape{} : ngraph::Shape{squeeze_indices.size()};
+    auto constant = std::make_shared<ngraph::opset1::Constant>(type, shape, squeeze_indices);
```

Comment:
I'm still wondering how it works when you pass `squeeze_indicces` e.g. {1,2} along with parameter isScalarAxes? Whether something works wrong or I don't understand how it's possible to construct such constant.